config/raft-quorum-1.properties

```diff
@@ -0,0 +1,24 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Not sure it's necessary to check these in.

Reply:
These properties make the raft server quorum setup easier.

Reply:
Sure, you could say the same thing for kafka's properties as well, but we only check in the standalone configuration. Similarly for zk. Not sure why we should do differently here.